inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/KafkaLoadNode.java

```diff
@@ -77,13 +93,57 @@ public String genTableName() {
     @Override
     public Map<String, String> tableOptions() {
         Map<String, String> options = super.tableOptions();
-        options.put("connector", "kafka");
         options.put("topic", topic);
         options.put("properties.bootstrap.servers", bootstrapServers);
-        options.put("format", format);
         if (getSinkParallelism() != null) {
             options.put("sink.parallelism", getSinkParallelism().toString());
         }
+        if (format instanceof JsonFormat) {
+            JsonFormat jsonFormat = (JsonFormat) format;
+            options.put("connector", "upsert-kafka");
+            options.put("key.format", jsonFormat.getFormat());
+            options.put("value.format", jsonFormat.getFormat());
+            options.put("value.json.fail-on-missing-field", jsonFormat.getFailOnMissingField() + "");
+            options.put("value.json.ignore-parse-errors", jsonFormat.getIgnoreParseErrors() + "");
+            options.put("value.json.timestamp-format.standard", jsonFormat.getTimestampFormatStandard());
+            options.put("value.json.map-null-key.mode", jsonFormat.getMapNullKeyMode());
+            options.put("value.json.map-null-key.literal", jsonFormat.getMapNullKeyLiteral());
+            options.put("value.json.encode.decimal-as-plain-number", jsonFormat.getEncodeDecimalAsPlainNumber() + "");
+            options.put("key.json.fail-on-missing-field", jsonFormat.getFailOnMissingField() + "");
+            options.put("key.json.ignore-parse-errors", jsonFormat.getIgnoreParseErrors() + "");
+            options.put("key.json.timestamp-format.standard", jsonFormat.getTimestampFormatStandard());
+            options.put("key.json.map-null-key.mode", jsonFormat.getMapNullKeyMode());
+            options.put("key.json.map-null-key.literal", jsonFormat.getMapNullKeyLiteral());
+            options.put("key.json.encode.decimal-as-plain-number", jsonFormat.getEncodeDecimalAsPlainNumber() + "");
+        } else if (format instanceof AvroFormat) {
```

Comment:
It is recommended to decouple Format and Node, and try to avoid modifying Node when adding format support